new/analyse/determine-version.ts

```diff
@@ -63,6 +63,16 @@ export const determineVersion: Module = (env) => {
       throw new Error(errorMessage);
     }
 
+    // If package has been processed already, skip it
```

Comment:
This is not needed. It costs next to no time to simply recalculate all packages and we can get rid of a quite complicated block.